mypy/errors.py

```diff
@@ -790,17 +791,16 @@ def remove_duplicates(self, errors: List[ErrorTuple]) -> List[ErrorTuple]:
             i += 1
         return res
 
-    def save_baseline(self, file: Path) -> None:
-        """Create/update a file that stores all errors"""
-        if not self.all_errors:
-            if not file.exists():
-                return
-            file.unlink()
-            print("No errors, baseline file removed")
-            return
-        if not file.parent.exists():
-            file.parent.mkdir()
-        self.baseline = {
+    def initialize_baseline(self, errors: Dict[str, List[object]], sources: List[str]):
+        """Initialize the baseline properties"""
+        errors_ = cast(Dict[str, List[BaselineError]], errors)
```

Comment:
why `errors_`, can it have a more descriptive name? or can you just redefine `errors`

Reply:
Can't redefine, redefining is currently disabled :(